corehq/apps/hqwebapp/widgets.py

```diff
@@ -63,10 +63,6 @@ def render(self, name, value, attrs=None, choices=()):
         return mark_safe(output)
 
 
-class Select2(_Select2Mixin, forms.Select):
-    pass
```

Comment:
OOC, what was your process for finding something like this?  Did you grep for every reference you removed to see if it could now be deleted?